elements/campaign-brand-messaging/campaign-brand-messaging.js

```diff
@@ -0,0 +1,46 @@
+import {
+  registerElement,
+  BulbsHTMLElement,
+} from 'bulbs-elements/register';
+import { filterBadResponse, getResponseJSON } from 'bulbs-elements/util';
+import invariant from 'invariant';
+
+class CampaignProductShot extends BulbsHTMLElement {
+
+  handleRequestSuccess (data) {
+    if (!data.brand_messaging) {
+      console.log('Brand messaging not set on this campaign');
```

Comment:
If we're going to do messages like this, we should also log the specific element:

```
console.log('Brand messaging not set on this campaign', this);
```

So we can right click/reveal in elements panel.